src/textField/index.unit.tsx

```diff
@@ -235,3 +235,10 @@ it('Simulates a blur event.', () => {
   wrapper.find('input').simulate('blur')
   expect(onTextFieldBlur).toHaveBeenCalledTimes(1)
 })
+
+it('Be able to format the value', () => {
+  const wrapper = mount(
+    <TextField name="test" defaultValue="Hello" format={value => `${value} world`} />,
+  )
+  expect(wrapper.find('input').prop('value')).toBe('Hello world')
+})
```

Comment:
Can you add a second expect with a `setProps()` to test that it's also updating correctly?